### DISCUSSION TITLE
linter.enabled = false not working

### DISCUSSION THREAD

**Discussion by vaibhavdugar** [biomejs/biome#1792] [Q&A]
In my biome.json file, I've enabled linter as false but it is not working and still giving me errors. Can someone help me out please?
<img width="443" alt="image" src="https://github.com/biomejs/biome-vscode/assets/76567443/ae99f44e-01e5-4cfc-9465-f28db0e39e8d">

**Comment by nhedger** [comment]
Does this happen exclusively in the VS Code extension of also when running the CLI against your code?

Also, if you can provide a reproduction repo, that would help.

**Reply by vaibhavdugar** [reply]
It's not working both in the cli and vscode extension.
Sorry I cannot provide you the repo as I am working in a company but I can tell you that it's a mono repo and I have configured biome at the root level. And changing the enabled property of linter object as false in the biome.json file, I am still getting errors.

**Reply by nhedger** [reply]
I've moved this to the main repo.

Can you paste the output of `biome rage` ?

**Reply by vaibhavdugar** [reply]
<img width="290" alt="image" src="https://github.com/biomejs/biome/assets/76567443/6212bc31-f7db-49ba-8724-fd94e0ea2a96">

**Reply by ematipico** [reply]
WIthout a way to reproduce the issue, it's very difficult to understand what's the problem. The rage command says that the configuration is loaded successfully. 

Here's a couple of things you can do:
- use `biome rage --daemon-logs` and see if you can find some useful information about the configuration and the rules applied
- run the CLI with `--log-level=info` or `--log-leve-debug` and see if you can find some useful information to share with us

**Reply by vaibhavdugar** [reply]
How can we check that how much time is biome taking for linting so that we can compare it with the time taken by ESLint for linting and check if it is actually faster?

**Reply by ematipico** [reply]
Biome tells you at the very end how long it took to lint your project.

If you're looking for a benchmark tool, check hyperfine, that's what I use.

**Reply by vaibhavdugar** [reply]
I want to configure biome in a particular project, so how to configure it in project.json of that project
like
"lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["apps/ccx-consumer/**/*.{ts,tsx,js,jsx}"]
      }
    },
this is the code for configuration of es-lint in project.json of that project, how can we do the same for biome

**Reply by ematipico** [reply]
I believe we answered your question on Discord. 

Anyway, I don't use nix, you should seek support from people that know how to use the tool
